tests/src/Common/CoreCLRTestLibrary/Utilities.cs

```diff
@@ -67,6 +67,7 @@ public static bool Verbose
         public static bool IsLinux => RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
         public static bool IsMacOSX => RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
         public static bool IsWindows7 => IsWindows && Environment.OSVersion.Version.Major == 6 && Environment.OSVersion.Version.Minor == 1;
+        public static bool IsWinRTSupported => IsWindows && !IsWindows7;
```

Comment:
This will include Nano which probably doesn't have the WinRT subsystem. We should check that out.

Reply:
It looks like Nano has at least some of the WinRT subsystem according to [MSDN's list of Nano Server APIs](https://msdn.microsoft.com/en-us/library/mt588480(v=vs.85).aspx). I can guess that the binding test might need to be disabled on Nano since I don't think Nano would have XAML islands support.

Reply:
It won't have any UI stack for sure.

Reply:
I've disabled the binding test on Nano.